packages/react-component-library/src/components/Timeline/Timeline.tsx

```diff
@@ -158,19 +180,15 @@ export const Timeline: React.FC<TimelineProps> = ({
       startDate={startDate}
       today={today}
     >
-      <div className="timeline" data-testid="timeline" role="grid">
-        <div className={innerClasses}>
+      <StyledTimeline data-testid="timeline" role="grid">
+        <Inner hasSide={hasSide || hasTimelineSide}>
           {rootChildren}
-          <div
-            className="timeline__header" // Kept for backwards compatibility in downstream applications
```

Comment:
@jpveooys you are dependent on `timeline__header`? (maybe it needs a unit test if it stays)

Reply:
Only for CSS overrides (now) so it's the same as another other CSS class in that respect

Reply:
I have preserved the original BEM class names for internal skeleton markup in order to avoid breaking any overrides consumers may created. We'll continue to look for a solution that will allow us to remove these in the next major.